### ISSUE TITLE
Missing DNS option in dhcp.py

### ISSUE THREAD

**Issue by pbertera** [psychomario/PyPXE#77]
Hi,

seems that the DNS option has been forgotten to be implemented.

a line like:

````
response += self.tlvEncode(6, socket.inet_aton(self.dnsserver)) #DNS
````
Should be added in the craftOptions method

**Comment by psychomario** [comment]
This is a bug I was aware of and is fixed in #65, which I believe is due to be merged after #59

**Comment by psychomario** [comment]
Actually this is probably quite a large bug, so I will get it merged with #59. Thanks for catching this!
